future.tex

```diff
@@ -1,83 +1,8 @@
-claimed that nearly 25GWe of nuclear should be generated by 2025, but there hasn't been
-a fixed amount. After the government became more favorable towards nuclear in 2006, utilities
-began planning for new plants. Also, many different foreign utilities, from France (EDF),
-Russia (Rosatom), and China (China General Nuclear Group). This led up to a total proposed
-number of 13 units ($17,900$ MWe) by 2030, with a mixture of EPR, ABWR, and AP1000s \cite{noauthor_nuclear_2017-3}.
-
-\subsection{Belgium}
-Belgium currently has seven nuclear reactors ($5,943$ MWe), but with little government support
-for nuclear energy, all of them are expected to shutdown in 2025. The government
-said that its stance on nuclear phase-out "is now final". There are no known plans
-to build new reactors \cite{noauthor_nuclear_2017-4}.
-
-\subsection{Sweden}
-Sweden has nine reactors that generate 40\% of the country's electricity. 
-Sweden had four reactors that would close by 2020 due to decreasing profits,
-with the nuclear tax of .75 Euro cents/kWh, which is one-third of the operating
-cost. The country plans to shut down all reactors by 2050 \cite{noauthor_nuclear_2017-5}
-
-\subsection{Finland}
-Finland currently has four reactors (2700 MWe), two BWR and two VVER,
-with a fifth and sixth one (EPR of 1600 MWe, VVER of 1200 MWe, respectively) planned.
-The EPR is to start operation in 2018 and the VVER in 2024.
-Finland, the country closest to an operating permanent geological repository, has 
-a very optimistic view of nuclear energy \cite{noauthor_nuclear_2017-6}.
-
-\subsection{Bulgaria}
-Bulgaria currently has two nuclear reactors that provide 1926 MWe capacity. The
-government has been historically in favor of nuclear power, and uses Russian
-technology for its reactors, namely the VVER. The government attempted to build
-new plants in 1980, but it was aborted in 1991 due to lack of funds. Yet,
-the government planned to add another unit to the currently existing site.
-There are bids in progress, and the most probable type and vendor seems to be
-Westinghouse's AP1000 with a capacity of 950 MWe \cite{noauthor_nuclear_2016}.
-
-
-\subsection{Poland}
-Poland has traditionally been using coal for most of its electricity generation,
-but due to recent concerns of CO2 emissions and air quality, its government decided
-to invest on nuclear, and plans to construct 6,000 MWe capacity up till 2035. The government
-is yet to decide on which type from which vendor, but estimates the operation date to be
-2029 for the first 3,000 MWe, and 2035 for the other 3,000 MWe. \cite{noauthor_nuclear_2017-7}
-
-\subsection{Summary}
-With all the other countries only taking up a minute portion of the entire
-EU nuclear fleet, it is assumed that they will not have additional constructions,
-but will cease nuclear generation when their current operating reactors shut down.
 
 In summary, the current, and the future of nuclear energy in Europe is organized
```

Comment:
Now that the above has been deleted, this is now the beginning of the section. You should not start the discussion with the words "In summary". Try something like an introduction to why the heck you're talking about future european nuclear build projections in the first place. The text should answer questions like:

- What is the reader supposed to learn in this section? (Probably, the reader will learn what assumptions your simulations made about european nuclear deployment, why those assumptions were made, and why they should be considered valid.)
- What does it have to do with your results? (Something about these european nations is important to the question the simulations are seeking to answer.)